index.html

```diff
@@ -424,6 +424,7 @@ <h1>MAXIMIZE YOUR WIN</h1>
           <a href="/embr" style="color: #b8bb26">Embr</a>
         </p>
       </div>
+      <img src="static/images/bobi.jpg" id="bobi" style="width: 480px; height: 640px;">
```

Comment:
could we introduce a gallery component here?

Reply:
I have actually no idea what I'm doing and don't know a gallery component is :( you can edit it however you want mate

Reply:
a gallery component is a mechanism to browse a wide array of images

Reply:
Thanks for the reply richbert! After a bit of searching on the internet and talking with gpt i made a small gallery, hope it works.